app/assets/javascripts/qsets.js

```diff
@@ -84,6 +94,7 @@ function initQuestionFilter() {
     // expecting filterType == 'all', 'mine', or 'other'
     var anyQuestionsToDisplay = anyQuestions(filterType);
     if (!anyQuestionsToDisplay) showNoQuestionNotification(noQuestionsMessage[filterType]);
+    $('#quizAllButtonID').attr('disabled', !anyQuestionsToDisplay);
```

Comment:
Recall we discussed this, can't use id here, because there are actually two quiz all buttons (but one of them is hidden, depending on screen size). Indeed testing on small screen shows the quiz all is not disabled when you'd expect. I think switching the selector from `$('#quizAllButtonID')` to `$('.quiz-all')` might do it.